integrations/resend/src/index.ts

```diff
@@ -1,17 +1,43 @@
 import type { IntegrationClient, TriggerIntegration } from "@trigger.dev/sdk";
 import { Resend as ResendClient } from "resend";
+import { RequestError } from "@octokit/request-error";
```

Comment:
We shouldn't have a dependency on the GitHub octokit package here. 

Instead you should import the `ErrorResponse` from the resend package. Pretty sure it's exported.

Reply:
Hi @matt-aitken ,

I attempted to import ErrorResponse from the Resend package; however, I couldn't find it available within the Resend package.

Reply:
If you update the resend package to the latest, they now export `ErrorResponse`